### PR TITLE
Update Dependencies

### LINKED ISSUE #430: Update Dependencies (Security)
There are security findings in NATS.Client 0.11.0:

CVE-2017-0249
CVE-2019-0820
CVE-2018-8292

The two packages with the vulnerability are system.net.http and system.text.regularexpressions . These are transitive references that come in via netstandard.library.1.6.1.

https://www.whitesourcesoftware.com/vulnerability-database/CVE-2017-0249
https://www.whitesourcesoftware.com/vulnerability-database/CVE-2019-0820
https://www.whitesourcesoftware.com/vulnerability-database/CVE-2018-8292

### PR DESCRIPTION
**Pull request by ColinSullivan1**
Resolves #430 (as best we can).

Signed-off-by: Colin Sullivan <colin@synadia.com>

### CODE REVIEW

File: src/Tests/IntegrationTests/TestBasic.cs [hunk 1]
```diff
@@ -1965,7 +1965,6 @@ public void TestServersOption()
             var exception = Record.Exception(() => Context.ConnectionFactory.CreateConnection());
 
             Assert.IsType<NATSConnectionException>(exception);
-            Assert.Equal("timeout", exception.Message);
```
Comment (ColinSullivan1) [inline src/Tests/IntegrationTests/TestBasic.cs:1968]: This was flapping locally so removed the message check....

**Review by watfordgnf — APPROVED** [review]

File: src/Tests/IntegrationTests/TestBasic.cs (hunk 1 above)
Comment (jasper-d) [reply src/Tests/IntegrationTests/TestBasic.cs:1968]: Thats strange. Do you remember what the actual message was?

Comment (ColinSullivan1) [reply src/Tests/IntegrationTests/TestBasic.cs:1968]: It was "Failed to connect", with an InnerException of ExtendedSocketConnection - "Connection refused...".  TBH, I'm OK with this, it's more accurate and the exception type didn't change.

Comment (jasper-d) [reply src/Tests/IntegrationTests/TestBasic.cs:1968]: I don't care about the message. I'm just a bit concerned that the behaviour isn't deterministic, but I cannot reproduce it locally.

**Pull request merged**